libcodechecker/libhandlers/check.py

```diff
@@ -271,7 +272,19 @@ def __update_if_key_exists(source, target, key):
         if key in source:
             setattr(target, key, getattr(source, key))
 
-    output_dir = os.path.abspath(args.output_dir)
+    # If no output directory is given then the checker results go to a
```

Comment:
This should be handled at the analyze command.

Reply:
Also keep the user guide up to date.

Reply:
I'm just thinking, does it make sens to run analyze with a default temp directory and remove the results without a parse?
In that case you can not check the results, they will be deleted after the analysis.